01/app.js

```diff
@@ -1 +1,11 @@
-console.log('DOM');
\ No newline at end of file
+const comments = document.querySelector(
+  ".comments__item.comments__item--newest"
+);
+
+if (comments) {
+  const elements = comments.querySelectorAll("[data-info]");
+
+  if (elements) {
```

Comment:
Pusta tablica to jest prawda więc ten if nie jest tu potrzebny, jak już to `if (elements.length)`